tests/src/rules/extensions.js

```diff
@@ -1,9 +1,79 @@
 import { RuleTester } from 'eslint'
 import rule from 'rules/extensions'
 import { test, testFilePath } from '../utils'
+import { expect } from 'chai'
 
 const ruleTester = new RuleTester()
 
+describe('extensions buildProperties', function() {
```

Comment:
i'm confused what the purpose of these tests are. can the tests be exercised with ruleTester test cases, rather than hardcoding to things like `__test__`?

Reply:
Sorry, I deleted my previous comment by accident. I did it like that, because buildProperties is private function and needs to be tested.
So I guess it's a no go for this PR?

Reply:
in general, private things shouldn't be tested - in this plugin's case, everything should be tested via the RuleTester.

Reply:
I'm not sure why you're saying "private things shouldn't be tested ".

Reply:
But when you refactor internals and break something, you wish you had tests to catch that early.

Reply:
The tests should be on the public pieces, and should transparently exercise all the private pieces.

Any behavior of your private code that is not observable via your public code *should not be tested because it doesn't matter if it works or not*